javascripts/discourse/routes/user-portfolio.js

```diff
@@ -17,20 +15,21 @@ export default DiscourseRoute.extend({
     });
   },
 
-  model: function () {
+  model() {
     const filterParams = {};
+    const filterTags = settings.portfolio_tags.replace(/\|/g, "+");
+    let tagQuery = filterTags ? `?tags=${filterTags}+` : "";
```

Comment:
Does it have to be in the URL and not in `filterParams` like the other two params?

Reply:
Looks like the backend doesn't like encoded `+` characters… Let's leave it (w/ a small tweak)